SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1819,5 +1814,87 @@ private void TrySendReferenceNumberEmail(string emailAddress, string reference)
                 }
             }
         }
+        
+        private PathByActionArguments GetBackArgs(
+            QuestionFlowStep currentStep,
+            PropertyData propertyData,
+            QuestionFlowStep? entryPoint = null)
+        {
+            var backStep = questionFlowService.PreviousStep(currentStep, propertyData, entryPoint);
+            return GetActionArgumentsForQuestion(backStep, propertyData?.Reference, entryPoint);
+        }
+
+        private PathByActionArguments GetActionArgumentsForQuestion(QuestionFlowStep question, string reference = null, QuestionFlowStep? entryPoint = null)
+        {
+            return question switch
+            {
+                QuestionFlowStep.Start => new PathByActionArguments(nameof(Index), "EnergyEfficiency"),
+                QuestionFlowStep.NewOrReturningUser => new PathByActionArguments(nameof(NewOrReturningUser_Get), "EnergyEfficiency"),
+                QuestionFlowStep.OwnershipStatus => new PathByActionArguments(nameof(OwnershipStatus_Get), "EnergyEfficiency", GetPathValues(reference, entryPoint)),
```

Comment:
Some of these (for example Ownership Status) are unreachable with an entry point. Please double check the others.